### ISSUE TITLE
Compatibility with Electrum client 3.1.x - ElectrumX 1.3

### ISSUE THREAD

**Issue by 4oo4** [lukechilds/docker-electrumx#7]
Just tried upgrading my Electrum client to the latest 3.1.2, but got this error with `docker logs` about not having the right SSL protocol version (also happened with 3.1.0 and 3.1.1):

```
INFO:Controller:SSL server listening on :50002
INFO:ElectrumX:[0] SSL 10.10.99.105:50312, 1 total
INFO:ElectrumX:[0] unsupported protocol version request 1.2
INFO:ElectrumX:[0] unsupported protocol version: 1.2
```
Not sure if this is upstream or not?

Cheers

**Comment by lukechilds** [comment]
Definitely seems like an upstream problem to me. If your client can talk to the server then the Docker container is working.

It looks like Electrum 3.1.2 wants an electrum server that talks version 1.2 of the electrum protocol. Maybe ElectrumX 1.3 dropped support for this, or doesn't support it yet. Best raise the issue on the ElectrumX repo.

I'll close this issue now, but feel free to re-open you find evidence to suggest it is actually a problem with this Docker container.

**Comment by 4oo4** [comment]
Sounds good

https://github.com/kyuupichan/electrumx/issues/436

**Comment by lukechilds** [comment]
@4oo4 Btw, I just released `v1.4`. Might be worth checking that and seeing fi your issue still persists.

**Comment by 4oo4** [comment]
@lukechilds That fixed it. Thanks!
